frontend/Dockerfile

```diff
@@ -0,0 +1,19 @@
+# Use Node.js base image
+FROM node:latest
```

Comment:
Instead of using `node:latest`, consider using a slim node image like alpine will greatly slim down the size of the docker image from my experience.